ang/my-activities/app.routes.js

```diff
@@ -0,0 +1,13 @@
+(function (angular, $, _) {
+  var module = angular.module('my-activities');
+
+  module.config(function ($routeProvider) {
+    $routeProvider.when('/', {
+      reloadOnSearch: false,
+      resolve: {
+        hiddenFilters: function () {}
+      },
+      template: '<civicase-my-activities></civicase-my-activities>'
+    });
+  });
+})(angular);
```

Comment:
same question than before, but I see the parameters are not included here on the arguments but on the function definition.